src/_data/speakers.yml

```diff
@@ -62,7 +62,7 @@
   bio: |
     He(lio) has created things primarily in C/C++ & Ruby, over the past 20+ years.
     He first met Ruby at v1.8.7 and Rails 2.x in 2010 and he is a Happy Developer since then.
-    He is currently a part-time Rails consultant, while building a new product and doing some OSS!"
```

Comment:
I had to remove the trailing `"` here, it completely broke the layout because of the extra quote within the `alt` tag.